src/api/telemetry/BatchingWebSocket.js

```diff
@@ -20,25 +20,16 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 import installWorker from './WebSocketWorker.js';
+
+// Shim for Internet Explorer, I mean Safari. It doesn't support requestIdleCallback, but it's in a tech preview, so it will be available soon.
+const requestIdleCallback =
```

Comment:
Let's add some JSDocs around this polyfill.

We also should be failing our `browserlist` check on this. If we're not, we need to figure out why

Reply:
Copy, wilco!